io.github.ar_.apple-flinger.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.ar_.apple-flinger
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: run.sh
+rename-desktop-file: io.github.ar_.apple-flinger.desktop
```

Comment:
why do you need to rename? it already has the name.

Reply:
AFAIK it not only renames the file but replaces some fields in it which will be useful

Reply:
replace what?